test/fit_tests/requirements.txt

```diff
@@ -1,4 +1,11 @@
-requests==2.2.1
+gevent==1.1.2
+greenlet==0.4.10
+nose==1.3.7
 pexpect==3.1
 pika==0.10.0
-nose==1.3.7
+pkg-resources==0.0.0
```

Comment:
Hitting an error on the pkg-resources requirement when setting up my virtual test env.
```
Downloading/unpacking pika==0.10.0 (from -r requirements.txt (line 5))
  Downloading pika-0.10.0-py2.py3-none-any.whl (92kB): 92kB downloaded
Downloading/unpacking pkg-resources==0.0.0 (from -r requirements.txt (line 6))
  Could not find any downloads that satisfy the requirement pkg-resources==0.0.0 (from -r requirements.txt (line 6))
Cleaning up...
No distributions at all found for pkg-resources==0.0.0 (from -r requirements.txt (line 6))
Storing debug log for failure in /emc/hohene/.pip/pip.log
```